contracts/extensions/VotingReputation.sol

```diff
@@ -559,550 +258,4 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs {
-  /// @return The current motion state
-  function getMotionState(uint256 _motionId) public view returns (MotionState) {
-    Motion storage motion = motions[_motionId];
-    uint256 requiredStake = getRequiredStake(_motionId);
-
-    // Check for valid motion Id
-    if (_motionId == 0 || _motionId > motionCount) {
-
-      return MotionState.Null;
-
-    // If finalized, we're done
-    } else if (motion.finalized) {
-
-      return MotionState.Finalized;
-
-    // Not fully staked
-    } else if (
-      motion.stakes[YAY] < requiredStake ||
-      motion.stakes[NAY] < requiredStake
-    ) {
-
-      // Are we still staking?
-      if (block.timestamp < motion.events[STAKE_END]) {
-        return MotionState.Staking;
-      // If not, did the YAY side stake?
-      } else if (motion.stakes[YAY] == requiredStake) {
-        return MotionState.Finalizable;
-      // If not, was there a prior vote we can fall back on?
-      } else if (add(motion.votes[NAY], motion.votes[YAY]) > 0) {
-        return MotionState.Finalizable;
-      // Otherwise, the motion failed
-      } else {
-        return MotionState.Failed;
-      }
-
-    // Fully staked, go to a vote
-    } else {
-
-      if (block.timestamp < motion.events[SUBMIT_END]) {
-        return MotionState.Submit;
-      } else if (block.timestamp < motion.events[REVEAL_END]) {
-        return MotionState.Reveal;
-      } else if (
-        block.timestamp < motion.events[REVEAL_END] + escalationPeriod &&
-        motion.domainId > 1
-      ) {
-        return MotionState.Closed;
-      } else {
-        return MotionState.Finalizable;
-      }
-
-    }
-  }
-
-  /// @notice Get the voter reward
-  /// NB This function will only return a meaningful value if in the reveal state.
-  /// Prior to the reveal state, getVoterRewardRange should be used.
-  /// @param _motionId The id of the motion
-  /// @param _voterRep The reputation the voter has in the domain
-  /// @return The voter reward
-  function getVoterReward(uint256 _motionId, uint256 _voterRep) public view returns (uint256) {
-    Motion storage motion = motions[_motionId];
-    uint256 fractionUserReputation = wdiv(_voterRep, motion.repSubmitted);
-    uint256 totalStake = add(motion.stakes[YAY], motion.stakes[NAY]);
-    return wmul(wmul(fractionUserReputation, totalStake), voterRewardFraction);
-  }
-
-  /// @notice Get the range of potential rewards for a voter on a specific motion, intended to be
-  /// used when the motion is in the reveal state.
-  /// Once a motion is in the reveal state the reward is known, and getVoterRewardRange should be used.
-  /// @param _motionId The id of the motion
-  /// @param _voterRep The reputation the voter has in the domain
-  /// @param _voterAddress The address the user will be voting as
-  /// @return The voter reward
-  function getVoterRewardRange(uint256 _motionId, uint256 _voterRep, address _voterAddress) public view returns (uint256, uint256) {
```

Comment:
Resolved by providing a `getVoterRewardRange` which takes influence directly as an argument (an array, rather than a single uint256, to represent the possibility of multiple influences).